Formula/asymptote.rb

```diff
@@ -0,0 +1,39 @@
+class Asymptote < Formula
+  desc "Powerful descriptive vector graphics language"
+  homepage "https://asymptote.sourceforge.io"
+  url "https://downloads.sourceforge.net/project/asymptote/2.61/asymptote-2.61.src.tgz"
+  sha256 "f3ccaa6fde02a2d37fca771563597828530f94821e1c3fe3a5c7d81e7d398d3d"
+
+  depends_on "fftw"
+  depends_on "ghostscript"
+  depends_on "glm"
+  depends_on "gsl"
+  depends_on "readline" # Remove in next release to use macOS editline
+
+  resource "manual" do
+    url "https://downloads.sourceforge.net/project/asymptote/2.61/asymptote.pdf"
+    sha256 "b5f5848515b6fd6e6d70cebc1b33c45e9d0f00ab7fcf50f71f95e7748c98f5da"
+  end
+
+  def install
+    touch buildpath/"doc/asy-latex.pdf"
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
Will do